pages/TheExplorationEditor.html

```diff
@@ -38,8 +38,8 @@
   </h3>
 
   <p>
-    A card consists of three parts: some 'content', an 'interaction' (or
-    question), and 'responses' that specify what Oppia should do next, based on
+    A card consists of three parts: some <em><strong>content</strong></em>, an <em><strong>interaction</strong></em> (or
```

Comment:
Ah my fault, I made an example to show you on the thread of issue #1614, then I came back to fix the Home page and forgot to reset this back back to strong. Sorry!